web/src/components/LandingPageTiles.tsx

```diff
@@ -25,67 +29,95 @@ export const LandingPageTiles = (): ReactElement => {
     });
   };
 
+  interface ActionTileProps {
+    imgPath: string;
+    tileText: string;
+    tileText2?: string;
+    dataTestId: string;
+    onClick: () => void;
+    isPrimary?: boolean;
+  }
```

Comment:
We define this same interface in 2 places. Here and in `LandingPageSlider`. The `Props` interface in `LandingPageActionTile` is also very similar to this shape.

I don't think we're necessarily using this enough to warrant it being added to `types.ts`, but maybe we could cut down on the duplication by renaming and exporting the `Props` interface in `LandingPageActionTile`.

Reply:
I'd actually suggest putting it in `types` - I think it's generally an antipattern to have components importing things from other components other than the React Component itself.

Reply:
One reason for this is that there is no real reason for one component file to "own" the type definition. If both  `LandingPageTiles` and `LandingPageSlider` use this interface, why should LandingPageSlider own it and define it? And if someone is looking for that type definition, why would they expect it to be in Slider as opposed to the Tiles or anywhere else? Having a single `types` as a shared place for shared types addresses all this

Reply:
I would also rename the type to perhaps just `ActionTile`? It's an object that defines a tile, and "props" is a specific word for component props that doesn't really fit what's going on here.